### ISSUE TITLE
[1.7.7] Mismatch between imagestreams in fuse-online-install and operator template

### ISSUE THREAD

**Issue by avano** [syndesisio/fuse-online-install#149]
fuse-online-install specifies the image stream for data virt as `fuse-ignite-komodo` and the tag as `1.7`:

https://github.com/syndesisio/fuse-online-install/blob/5032f8374613eeaea0a15e817006efdf2700c8a6/resources/fuse-online-image-streams.yml#L126-L143

however the internal template in the operator expects:

```
from:
          kind: ImageStreamTag
          name: fuse-komodo-server:latest
```

There are two issues logged for that: https://issues.jboss.org/browse/ENTESB-11036 https://issues.jboss.org/browse/ENTESB-11037

So either those two will be scheduled for CR2, or we need to edit the resources in fuse-online-install.

**Comment by heiko-braun** [comment]
@nicolaferraro I would suggest we update `fuse-online-install/resources/fuse-online-image-streams.yml`

**Comment by nicolaferraro** [comment]
Fixed by #151
